docs/proposals/mcs/remove-derived-mcs.md

```diff
@@ -0,0 +1,109 @@
+# Remove "derived-" prefix of MCS 
+
+## Summary
+
+As mentioned in [issue-2384](https://github.com/karmada-io/karmada/issues/2384), currently karmada uses ServiceExport, ServiceImport and other CR to form mcs-controller and manage multi-cluster services. But in the cluster where the service is not located, we need to use a different IP or a service name with a "derived-" prefix to access. We hope to reduce this difference so that different clusters can use the same domain name to access the same service.
+
+## Motivation
+
+Currently, in different cluster members of Karmada, if you need to access the same service across clusters, you need to use different ip or domain names with a "derived-" prefix to access. This will increase the complexity of the user's use. We hope that users can use the same domain name to achieve cross-cluster access.
+
+### Goals
+
+* Keep the original access method. Avoid the need to make modifications to the customer's system after the upgrade.
+* Added the use of a unified domain name for cross-cluster access. The domain name is the access domain name of the cluster where the service resides.
+
+### Non-Goals
+
+none
+
+## Proposal
+
+### User Stories
+
+According to the current Karmada's  [mcs configuration](https://karmada.io/docs/userguide/service/multi-cluster-service) , if the service is currently in member1, the user needs to access the service in member1, needs to use the domain name: "serve.default.svc.cluster.local", and the domain name in member2 will be: "derived-serve.default.svc.cluster.local". As a result, when customers use the service, they need to add the prefix "derived-" to access the domain name of the service. This brings additional inconvenience. So I hope to be able to:
+
+1. Provide a unified access method across clusters.
+
+2. The original access method remains unchanged. So that existing business can also work normally.
+
+### Notes/Constraints/Caveats (Optional)
+
+1. The coredns version must be v1.9.3 or above, otherwise the multicluster feature is not supported
```

Comment:
Which version of Kubernetes started using CoreDNS v1.9.3？

Reply:
> Which version of Kubernetes started using CoreDNS v1.9.3？

yea, i will add the k8s version. The matching k8s verison is at v1.21.0 or above,